schematic/visualization/tangled_tree.py

```diff
@@ -59,19 +60,26 @@ def __init__(
         # Parse schema name
         self.schema_name = path.basename(self.path_to_json_ld).split(".model.jsonld")[0]
 
+        parsed_data_model = None
+
         # Instantiate Data Model Parser
-        data_model_parser = DataModelParser(
-            path_to_data_model=self.path_to_json_ld,
-        )
+        if data_model_graph_pickle is None:
+            data_model_parser = DataModelParser(
+                path_to_data_model=self.path_to_json_ld,
+            )
+
+            # Parse Model
```

Comment:
nit: This comment does not add additional context for the reader, consider removing.